SIGS/app/views/school_rooms/index.html.erb

```diff
@@ -23,72 +23,83 @@
 <% if @my_school_rooms.count == 0 %>
   <h3>Não há turmas cadastradas no momento.</h3>
 <% else %>
-<%= will_paginate @my_school_rooms, renderer: BootstrapPagination::Rails %>
-
-  <table class="table table-hover">
-    <tr>
-      <th>Disciplina</th>
-      <th>Turma</th>
-      <th>Cursos</th>
-      <th>Nº de Vagas</th>
-      <% if permission[:level] == 1 %>
-        <th style="text-align:center">Alocar / desalocar</th>
-        <th style="text-align:center">Solicitar alocação</th>
-        <th style="text-align:center">Alterar</th>
-        <th style="text-align:center">Excluir</th>
-      <% end %>
-    </tr>
+  <div class="">
```

Comment:
Delete div 
```suggestion

```

Reply:
The div's class was deleted

Reply:
I just checked here and the class on div was deletes on commit 0589d84, i will request to @mlfaa  the re-request review